src/common/Formatter.cc

```diff
@@ -296,6 +296,17 @@ void JSONFormatter::finish_pending_string()
   }
 }
 
+void JSONFormatter::add_value(std::string_view name, double val) {
```

Comment:
wat to add a test to https://github.com/ceph/ceph/blob/main/src/test/common/test_json_formatter.cc ?

Reply:
Added the test.